### PR TITLE
Create sailfish-qrshare-pt_PT.ts

### PR DESCRIPTION
**Pull request by amaretzek**

### CODE REVIEW

File: shareplugin/translations/sailfish-qrshare-pt_PT.ts [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+    </message>
+    <message id="qrshare-placeholder-qrcode-too_long">
+        <source>Text is too long for QR code.</source>
+        <extracomment>Placeholder text</extracomment>
+        <translation>Texto demasiado longo para código QR.</translation>
+    </message>
+    <message id="qrshare-placeholder-aztec-too_long">
+        <source>Text is too long for Aztec code.</source>
+        <extracomment>Placeholder text</extracomment>
+        <translation type="unfinished">Texto demasiado longo para código Aztec.</translation>
```
Comment (monich) [inline shareplugin/translations/sailfish-qrshare-pt_PT.ts:29]: I leave `"unfinished"` for the strings that haven't been verified by a native speaker (or not translated at all). This `"unfinished"` should't be here but I can remove it myself, no problem.

**Comment by monich** [comment]
Merged, thanks!

File: shareplugin/translations/sailfish-qrshare-pt_PT.ts (hunk 1 above)
Comment (amaretzek) [reply shareplugin/translations/sailfish-qrshare-pt_PT.ts:29]: So, who is your native speaker for pt_PT? me?? ;)

Comment (monich) [reply shareplugin/translations/sailfish-qrshare-pt_PT.ts:29]: I was assuming so!

**Pull request merged**